packages/portal/README.md

```diff
@@ -199,6 +199,61 @@ export default function Org({ variables }) {
 }
 ```
 
+#### i18n configuration
```

Comment:
@steveoni I think this section should be sibling of "Data fetching" section and should come afterwards.